jung-algorithms/src/test/java/edu/uci/ics/jung/layout/algorithms/FRBHIteratorLayoutAlgorithm.java

```diff
@@ -35,12 +35,12 @@
   @Override
   public void visit(LayoutModel<N> layoutModel) {
     super.visit(layoutModel);
-    tree = new BarnesHutQuadTree(layoutModel);
+    tree = new BarnesHutQuadTree(layoutModel.getWidth(), layoutModel.getHeight());
```

Comment:
If you don't think that we're ever going to use the Iterator forms of the BH algorithms, feel free to go ahead and delete them.  Comments you've made in the past suggest that the visitor design is strictly better; am I mistaken?

Reply:
That is why I moved them to the src/test/java.
The serve as a way to make sure that changes in the way the BHQT is visited do not break anything.

Reply:
Ah, I'd missed the package change.

I don't plan on leaving them in a release, though.

Reply:
not a package change, they are in the src/**test**/java directory instead of the src/**main**/java directory. As it is now, they will not be part of the release jar, but will be in the test-jar (if the test jar is published).
I'll repeat what I said before: "I think you should remove test code as soon as you are positive that everything works and that nobody will ever change it"

Reply:
Sorry, that's what I meant.  

I don't remember that comment (there have been many) but I believe that you said it, and (generally) agree with it.  In this case it seems like we might still want the test code to validate that the (visitor) BH and non-BH versions still both work in the face of future changes, even if we get rid of the Iterator BH version, but that's clearly outside the scope of this PR.